.goreleaser.yml

```diff
@@ -0,0 +1,42 @@
+before:
+  hooks:
+    - go mod download
+builds:
+  -
+    ldflags:
+      - -s -w -X "github.com/sonatype-nexus-community/nancy/buildversion.BuildVersion={{.Version}}"
+      - -s -w -X "github.com/sonatype-nexus-community/nancy/buildversion.BuildTime={{time "2006-01-02T15:04:05Z07:00"}}"
+      - -s -w -X "github.com/sonatype-nexus-community/nancy/buildversion.BuildCommit={{.FullCommit}}"
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+
+archives:
+  - 
+    name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+    format_overrides:
+      - goos: windows
+        format: zip
+
+checksum:
+  name_template: "{{ .ProjectName }}checksums.txt"
+  algorithm: sha256
+
+snapshot:
+  name_template: "{{ .Tag }}-next-{{.FullCommit}}"
```

Comment:
Does this pull in the previous tag value?

Reply:
It pulls in the current tag value. The way GoReleaser works, is someone creates a tag, then CircleCI would start a build and release it. I preferred this a tiny bit to what we had before because it makes doing major and minor releases possible. Goreleaser also has a ton of functionality related to Docker, rpms, etc... that we could leverage.